README.md

```diff
@@ -10,13 +10,41 @@ cargo install stellar-contract-cli --git https://github.com/stellar/stellar-cont
 
 ## Usage
 
+All values passed to `--arg` are the JSON representation of SCVals.
+
+### Directly invoking a function in a user specified WASM contract
+
 ```
-stellar-contract-cli invoke --file <WASMFILE> --fn <FUNCNAME> --arg i32:1 --arg i32:2
+stellar-contract-cli invoke --file <WASMFILE> vm-fn --fn <FUNCNAME> --arg '{"i32":32}' --arg '{"i32":4}'
 ```
 
 Example using the [example_add_i32](https://github.com/stellar/rs-stellar-contract-sdk/tree/main/examples/add_i32) contract:
 
 ```
-$ stellar-contract-cli invoke --file example_add_i32.wasm --fn add --arg i32:1 --arg i32:2
-i32:3
+$ stellar-contract-cli invoke --file example_add_i32.wasm --fn add --arg '{"i32":32}' --arg '{"i32":4}'
```

Comment:
If it is low effort to revert the strval change and move it to a separate PR, I think we should address that change in a separate PR so that the majority of the changes in this PR are not blocked by #22. This format is close, but not exactly what I think @paulbellamy was proposing, as he was proposing `--arg 32`.

Reply:
Spoke to @paulbellamy who said he can work from here with #22, so let's merge this as is.